app/configurator/components/chart-options-selector.tsx

```diff
@@ -494,6 +490,15 @@ const EncodingOptionsPanel = (props: EncodingOptionsPanelProps) => {
                 }
               />
             ) : null}
+            {encoding.options?.adjustScaleDomain && fieldComponent ? (
+              <ChartScaleDomain
+                chartConfig={chartConfig}
+                field={field}
+                defaultDomain={encoding.options.adjustScaleDomain.getDefaultDomain(
+                  { chartConfig, observations }
+                )}
```

Comment:
Should the default domain be memoed ? I presume we need to go over all observations to compute the extent ?

Reply:
It could possibily be done in the ChartScaleDomain component.